conans/model/conf.py

```diff
@@ -108,7 +108,6 @@
     "tools.apple:enable_visibility": "(boolean) Enable/Disable Visibility Apple Clang flags",
     "tools.env.virtualenv:powershell": "If it is set to True it will generate powershell launchers if os=Windows",
     # Compilers/Flags configurations
-    "tools.build.cross_building:cross_build": "Decides whether cross-building or not regardless of arch/OS settings. Used by 'conan.tools.build.cross_building' function.",
```

Comment:
@memsharded It was here. Now, it's above.

Reply:
Oh, you are right! Sorry I didn't see it.